packages/cxl-ui/src/components/cxl-filter-header.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { ResizeController } from '@lit-labs/observers/resize-controller.js';
+import { customElement, state, query } from 'lit/decorators.js';
+import './cxl-vaadin-accordion';
+import '@vaadin/button'
+import cxlDashboardFilterHeaderStyles from '../styles/cxl-filter-header-css.js';
+
+const supportsScrollEndEvent = 'onscrollend' in window
+const isFirefox = document.scrollingElement.scrollLeftMax  !== undefined
```

Comment:
What's with Firefox that we need special flag?

Reply:
When in smaller screens, I implemented scrolling for the tabs, like in the design. I also implemented scroll snapping, so that it aligns to the start of the closest tab when the user stops scrolling. In Firefox, the smooth scrolling behavior causes a bug that prevents it from scrolling further on very specific screen sizes and tab widths, so I'm just using that to disable smooth scrolling in that case